js/utils/getBitcoinPrice.js

```diff
@@ -13,107 +13,115 @@ module.exports = function(currency, callback){
   if(currency !== 'BTC')
   {
 
+    window.btcAverages = {};
     var btPrices = [];
+    var btcAverages = {rates: {}};
 
-    //initial call
-    //bitcoin average is hitting the rate limit
-    /*
-  $.ajax({
-    method: "GET",
-    url: "https://api.bitcoinaverage.com/ticker/global/" + currency
-  })
+
+    var callBlockchain = function ()
+    {
+      $.ajax({
+        method: "GET",
+        url: "https://blockchain.info/ticker"
+      })
       .done(function (response)
       {
-        //console.log("bitcoinAverage: " + response['24h_avg']);
-        if($.isNumeric(response['24h_avg'])) {
-          btPrices.push(response['24h_avg']);
+        var blockChainCurrencies = {};
+        for (var currency in response) {
```

Comment:
On all of your for in loops, wrap the inside in an if statement that makes sure the properties belong to the object, not the object's prototype. It probably works fine in this case, but this is a JSHint check we require before code is merged. 

For example:

for (var blockChainCurrency in response) {
  if (response.hasOwnProperty(blockChainCurrency)) {
    ...do things
  }
}

https://jslinterrors.com/the-body-of-a-for-in-should-be-wrapped-in-an-if-statement